cobra/io/sbml.py

```diff
@@ -1241,7 +1243,7 @@ def _parse_notes_dict(sbase):
     """
     notes = sbase.getNotesString()
     if notes and len(notes) > 0:
-        pattern = r"<p>\s*(\w+\s*\w*)\s*:\s*([\w|\s]+)<"
+        pattern = r"p>\s*(\w+\s*\w*)\s*:\s*(.*?)</[^<>p]+p>"
```

Comment:
Can you please comment what this regex is supposed to match? I think making it non-greedy (`(.*?)`) is the right choice but the `</[^<>p]+p>` is unclear to me. You want to match until a non-paragraph element? The original regex also kinda smells so maybe we find a better solution? Also, this should probably be a compiled regex and match case-insensitively since HTML has no case requirement.

Reply:
I see now the relation to https://github.com/opencobra/cobrapy/issues/888 sorry.